### PR TITLE
Add type definitions for json-to-plain-text 

### PR DESCRIPTION
**Pull request by sumithemmadi**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/json-to-plain-text-tests.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// Import the module
```
Comment (jakebailey) [inline types/json-to-plain-text-tests.ts:1]: This file is in the wrong folder; I think it's just a duplicate of the other test file and needs to be removed.

**Review by jakebailey — CHANGES_REQUESTED** [review]

File: types/json-to-plain-text-tests.ts (hunk 1 above)
Comment (sumithemmadi) [reply types/json-to-plain-text-tests.ts]: I have removed the file.

File: types/json-to-plain-text/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    color?: boolean;
+    spacing?: boolean;
+    squareBracketsForArray?: boolean;
+    doubleQuotesForKeys?: boolean;
+    doubleQuotesForValues?: boolean;
+}
+
+declare function jsonToPlainText(data: unknown, options: Options): string;
+
+export { jsonToPlainText, Options };
```
Comment (jakebailey) [inline types/json-to-plain-text/index.d.ts:16]: Why not just `export` each of the two instead?

**Review by jakebailey — CHANGES_REQUESTED** [review]
Actually, this package is already in TS; why send these?

image

**Pull request closed without merging**